src/components/Recollect/index.js

```diff
@@ -18,7 +18,9 @@ class Recollect extends Component {
     );
     let loader = new window.Recollect.Widget.Loader(widgetOptions);
     const scrollToId = window.location.hash.split('#')[1];
-    document.getElementById(scrollToId).scrollIntoView(true);
```

Comment:
I caught this additional bug while doing this fix. Sometimes, when going to a guide page *without* a specific scroll-to point in the guide, you'll get an error `cannot scrollTo of undefined`. This catches this case when it draw the error and won't auto-scroll.